### PR TITLE
Added Dockerfile

### PR DESCRIPTION
**Pull request by rkcribber**
Dockerfile uses ubuntu16

### CODE REVIEW

**Comment by rkcribber** [comment]
Solves https://github.com/anmol27katyani/Public-API-crawler/issues/2

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,17 @@
+FROM ubuntu:16.04
```
Comment (anmol27katyani) [inline Dockerfile:1]: Why not use latest version?

Comment (rkcribber) [reply Dockerfile:1]: Latest version is less stable than 16.04, If you want I can use it

Comment (anmol27katyani) [reply Dockerfile:1]: No that makes sense. Thanks

**Pull request merged**